### DISCUSSION TITLE
IntelliSense and linux user question

### DISCUSSION THREAD

**Discussion by ToasterSticks** [coder/code-server#3055] [Q&A] [answered]
Hey, I have a question 😊 

~~1) Is it normal for IntelliSense to not work with TS files out of the box, or am I missing something?~~ Issue was resolved after a reload

2) Is it possible to switch the default username from "coder", to something else? If so, how?

**Comment by nooobcoder** [comment, marked as answer]
@ToastOnAStik You might face user permission issues by adding an user (not guaranteed)

`sudo adduser username`

### To give sudo rights to the user,

`sudo usermod -aG sudo username`

### Follow below screenshots now,
![image](https://user-images.githubusercontent.com/50350828/113811270-8275bd00-9789-11eb-9df5-56724c261fa9.png)

### To switch to the newly created user, use 
`su username`

![image](https://user-images.githubusercontent.com/50350828/113811328-a5a06c80-9789-11eb-9730-1343b5a2e2f8.png)


### If you need to delete the newly created user,
`sudo userdel ubuntu`

![image](https://user-images.githubusercontent.com/50350828/113811686-4d1d9f00-978a-11eb-89c8-7914baceb09c.png)

**Reply by nooobcoder** [reply]
@ToastOnAStik  please mark the discussion as answered, if I have cleared your query, feel free to leave back any questions if you are stuck. 
**code-server runs on a debian container** so most of the commands can be executed as you would usually do on Ubuntu/ other debian distros.

**I dont think creating another user would really benefit you**, because you might get into file perm issues at some point of time. Also you cannot change the root password of code-server.
